taxonomies/categories.txt

```diff
@@ -30384,6 +30697,13 @@ it:Tiramisù
 nl:Tiramisu
 wikidata:en:Q131582
 
+<en:Tiramisu
+en:Refrigerated tiramisu
+fr:Tiramisu rayon frais
+ciqual_food_code:en:19698
+ciqual_food_name:en:Tiramisu, refrigerated
+ciqual_food_name:fr:Tiramisu, rayon frais
```

Comment:
Unless we have a more generic Ciqual value for other kinds of Tiramisu, I think we should move the ciqual properties directly to en:Tiramisu. This is because we already have many products under Tiramisu.

Reply:
Thanks, fixed on next commit